app/views/home/_team.html.erb

```diff
@@ -5,165 +5,32 @@
-       </div>
-
-       <div class="card-reveal">
-         <span class="card-title white-text">
-           <h5>Diego Marczal</h5>
-           <h6>Idealizador da FARMA <br /> Desenvolvimento Web</h6>
-         </span>
-         <p>
-           Professor Adjunto da Universidade Tecnológica Federal do Paraná -
-           Câmpus Guarapuava
-         </p>
-         <a href="http://diego.marczal.com" class="right"><b>SAIBA MAIS</b></a>
-       </div>
-     </div>
-   </div>
-
-   <div class="col s12 m4"> <!-- MEMBRO 02  -->
-     <div class="card hoverable">
-       <div class="card-image waves-effect waves-block waves-light">
-         <%= image_tag("team/farma-02.jpg", class: "activator") %>
-       </div>
-       <div class="card-reveal">
-         <span class="card-title white-text">
-           <h5>Eleandro Maschio</h5>
-           <h6>Inteligência Artificial aplicada à Educação</h6>
-         </span>
-         <p>
-           Professor Adjunto da Universidade Tecnológica Federal do Paraná -
-           Câmpus Guarapuava
-         </p>
-         <a href="http://lattes.cnpq.br/5642072550423634" class="right"><b>SAIBA MAIS</b></a>
-       </div>
-     </div>
-   </div>
-
-   <div class="col s12 m4"> <!-- MEMBRO 03  -->
-     <div class="card hoverable">
-       <div class="card-image waves-effect waves-block waves-light">
-         <%= image_tag("team/farma-03.jpg", class: "activator") %>
-       </div>
-
-       <div class="card-reveal">
-         <span class="card-title white-text">
-           <h5>Alex S. de Castilho</h5>
-           <h6>Matemática</h6>
-         </span>
-         <p>
-           Professor Assistente da Universidade Tecnológica Federal do Paraná -
-           Câmpus Guarapuava
-         </p>
-         <a href="http://lattes.cnpq.br/0971381050120580" class="right"><b>SAIBA MAIS</b></a>
-       </div>
-     </div>
-   </div>
-
-   <div class="col s12 m4"> <!-- MEMBRO 04  -->
+  <% @developers.each do |dev| %>
+   <div class="col s12 m4">
      <div class="card hoverable">
        <div class="card-image waves-effect waves-block waves-light">
-         <%= image_tag("team/farma-04.jpg", class: "activator") %>
+         <%= image_tag attachment_url(dev, :profile_image, :fill, 300, 300, format: "jpg", fallback: "default-avatar"), :class => 'circle index-img activator' %>
        </div>
-
        <div class="card-reveal">
          <span class="card-title white-text">
-           <h5>Jefferson H. Ramos</h5>
-           <h6>Desenvolvedor Web</h6>
+           <h5> <%= dev.name %> </h5>
+           <h6> <%= dev.function %> </h6>
          </span>
          <p>
-           Acadêmico do curso de Tecnologia em Sistemas para Internet da Universidade
-           Tecnológica Federal do Paraná - Câmpus Guarapuava.
+          <%= dev.institution %>
          </p>
-         <a href="http://lattes.cnpq.br/8918743248807944" class="right"><b>SAIBA MAIS</b></a>
+         <b><%= link_to 'SAIBA MAIS', "http://#{dev.link}", class: 'right' %></b>
```

Comment:
This can result in many erros, for exemple if my link is https, it will not work.

You need to do the right validation at the model, and just use the link on view.